tests/Integration/OtherTest.php

```diff
@@ -247,4 +247,50 @@ public function test_post_has_trackable_status_password_protected_with_filter():
 		$result = Parsely::post_has_trackable_status( $post );
 		self::assertTrue( $result );
 	}
+
+	/**
+	 * Tests if keywords in Parse.ly metadata are generated correctly when categories are tags.
+	 *
+	 * @since 3.0.3
+	 *
+	 * @covers \Parsely\Parsely::construct_parsely_metadata
+	 * @uses \Parsely\Parsely::get_categories
+	 */
+	public function test_post_with_categories_as_tags(): void {
```

Comment:
Since the structured data output is being tested here, then putting these tests in SinglePostTest.php would seem a better fit, and we already have some tests there that may cover things: https://github.com/Parsely/wp-parsely/blob/develop/tests/Integration/StructuredData/SinglePostTest.php#L172-L196